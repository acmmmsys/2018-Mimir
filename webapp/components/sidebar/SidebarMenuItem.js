import React, {Component} from 'react';
import classnames from 'classnames';
import 'style/sidebar/SideBar.scss';

class SidebarMenuItem extends Component {
    render() {
        return (
            <li className='ui-sidebar-menu-item'>
                { this.props.children }
            </li>
        )
    }
}

export default SidebarMenuItem;