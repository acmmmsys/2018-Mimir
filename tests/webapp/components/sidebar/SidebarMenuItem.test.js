import React from 'react';
import SidebarMenuItem from '../../../../webapp/components/sidebar/SidebarMenuItem'
import renderer from 'react-test-renderer';

let wrapper;

test('SidebarMenuItem does render', () => {
    wrapper = shallow(
        <SidebarMenuItem>

        </SidebarMenuItem>
    );
    expect(wrapper).toMatchSnapshot();
});
