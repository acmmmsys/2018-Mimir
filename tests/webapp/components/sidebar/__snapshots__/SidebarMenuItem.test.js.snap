// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SidebarMenuItem does render 1`] = `<li />`;
